routes/users.js

```diff
@@ -0,0 +1,9 @@
+const express = require('express');
+const router = express.Router();
+
+/* GET users listing. */
+router.get('/', function(req, res, next) {
+  res.send('respond with a resource');
+});
+
```

Comment:
From what ive read, it doesnt look like you currently require express or a database.